test/unit/test_mpfa_gravity.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Feb 28 20:55:56 2016
+
+@author: keile
+"""
+
+import numpy as np
+import scipy
+import sympy
+import unittest
+import porepy as pp
+
+# Analytical solution - hydrostatic pressure
+# with stepwise gravity variation
+x, y = sympy.symbols('x y')
+
+g1 = 10 
+g2 = 1
+
+p0 = 1 #reference pressure
+
+p = p0 + sympy.Piecewise(((1-y)*g1, y>=0.5), (0.5*g1+(0.5-y)*g2, y<0.5))
+gx = sympy.diff(p, x)
+gy = sympy.diff(p, y)
+
+p_f = sympy.lambdify((x, y), p, 'numpy')
+gx_f = sympy.lambdify((x, y), gx, 'numpy')
+gy_f = sympy.lambdify((x, y), gy, 'numpy')
+
+deviation_from_plane_tol=1e-5-5
```

Comment:
I think it would be better to put all of these "global variables" in the __init__ of the test class